ONSdigital/response-operations-ui
# Design system update

# What and why?
Upgrading design system to latest version at time of writing (61.0.4). There was a small breaking change introduced in version 60 where certain components associated with `pageConfig` needed to be moved into `pageConfig.header`.

Unrelated to the design system upgrade, I also noticed that the monochrome ONS logo wasn't appearing at the bottom of the page like it does on frontstage. This was because invalid parameters were being put in the footer. I removed these, which causes the ONS logo to appear properly.

# How to test?
* Deploy to environment and run acceptance tests with `make helm`. Note that you will need to set the `appVersion` in `values.yaml` to `pr-108` so that it uses the acceptance test fix I made. If not, you'll receive errors about the tests not being able to find certain elements on the page.
* Test out response-ops navigation and UI. There might be a few instances of elements changing size ever so slightly, but nothing significant should have changed.

# Trello
[Card](https://trello.com/c/hIMCYrXm)
